Command/GenerateDtoCommand.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace WsSys\DtoGeneratorBundle\Command;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console;
+use Symfony\Component\HttpKernel\Bundle\BundleInterface;
+use Sensio\Bundle\GeneratorBundle\Command\GeneratorCommand;
+use WsSys\DtoGeneratorBundle\Generator\XsdToDtoGenerator;
+
+/**
+ * Command to generate the Dtos
+ */
+class GenerateDtoCommand extends GeneratorCommand implements ContainerAwareInterface
+{
+    /**
+     * All the supported Types eg: xsd, json
+     * @var array 
+     */
+    protected $supportedTypes = array('xsd');
+    
+    /**
+     * @var ContainerInterface 
+     */
+    protected $container;
+    
+    /**
+     * @see Console\Command\Command
+     */
+    protected function configure()
+    {
+    	$this
+        ->setName('generate:dto')
+        ->setDescription('Convert Xml/Json into PHP.')
+        ->setDefinition(array(
+            new InputArgument(
+                'src', InputArgument::REQUIRED, 'The path of the souce XSD or JSON.'
+            ),
+            new InputArgument(
+                'destination', InputArgument::REQUIRED, 'The path to save DTOs.'
+            ),
+            new InputArgument(
+                'destinationNS', InputArgument::REQUIRED, 'The target namespace for DTOs'
+            ),
+            new InputArgument(
+                'type', InputArgument::REQUIRED, 'The path to save DTOs.'
```

Comment:
The description for the argument doesn't match the name - ```type```. It is a good idea to list the available types from the configuration.